### PR TITLE
Major updates 'n stuff

### PR DESCRIPTION
**Pull request by TheCurle**
So yeah. A lot happened recently.

### CODE REVIEW

**Review by ravenbuilder934 — APPROVED** [review]

**Comment by SizableShrimp** [comment]
My username is SizableShrimp, not Siz**e**ableShrimp as used in the quote data.

**Comment by ravenbuilder934** [comment]
Needs the latest quotes!

**Comment by TheCurle** [comment]
You needs the latest stabbies!

**Comment by ravenbuilder934** [comment]
Okay fine, but *also* the latest quotes!

File: quotes/data.json [hunk 1]
```diff
@@ -332,13 +334,21 @@
...
+            "name": "Collaborator"
+        },
         "docs_triage": {
             "name": "Docs Triage"
         },
         "superhelper": {
-            "color": "#c27c0e",
+            "color": "#ee5beb",
             "name": "superhelper"
         },
```
Comment (sciwhiz12) [inline quotes/data.json:347]: `superhelper` is above `collaborator`, according to the roles list (as viewed through Discord Mobile).

File: quotes/data.json [hunk 2]
```diff
@@ -332,13 +334,21 @@
...
+        "retrogradle": {
+            "color": "#15e4c7",
+            "name": "Retrogradle"
+        },
```
Comment (sciwhiz12) [inline quotes/data.json:351]: `retrogradle` is below `contributor`, according to the roles list (as viewed through Discord Mobile).

File: quotes/data.json [hunk 3]
```diff
@@ -1586,6 +1600,284 @@
...
+            "user": "Lex",
+            "text": "I hate you all...Everyone involed [sic] in that travesty"
+        },
+        {
+            "user": "Amethyst",
+            "text": "commit toaster bath is my new favorite phrase"
+        },
+        {
+            "user": "FinnT730",
+            "text": "<tt>pmm</tt> Porn Mod Manager"
```
Comment (sciwhiz12) [inline quotes/data.json:1775]:
```suggestion
            "text": "<code>pmm</code> Porn Mod Manager <code>/S</code>"
```

File: quotes/data.json [hunk 4]
```diff
@@ -1586,6 +1600,284 @@
...
+            "user": "Amethyst",
+            "text": "Damn right I'm beating you. Like an egg."
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "hey look I made a dick"
+        },
+        {
+            "user": "Amethyst",
+            "text": "then you realise the truth [...] there is no spoon."
```
Comment (sciwhiz12) [inline quotes/data.json:1807]:
```suggestion
            "text": "then you realise the truth <p>there is no spoon.</p>"
```

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]
Needs more changes before we can :shipit:

**Comment by TheCurle** [comment]
Superhelper above Collaborator, Retrogradle below Contributor.
tt -> code, [..] -> <p>

I'll poke the newest quotes in a few minutes. The mobile interface is such a gigantic pain in the repository.

**Comment by FinnT730** [comment]
> I'll poke the newest quotes in a few minutes.

It's been a full day, dear

**Comment by ravenbuilder934** [comment]
*23 hours

**Comment by FinnT730** [comment]
> *23 hours

You lying bird! /S
image

**Comment by ravenbuilder934** [comment]
Github *rounds* you see
You must take a left at the Wendy's, go down Main, and hover over the "yesterday"

**Comment by TheCurle** [comment]
THERE.
IT'S DONE.
ARE YOU HAPPY NOW?

File: quotes/data.json [hunk 5]
```diff
@@ -1878,6 +1888,197 @@
...
+            "user": "illy",
+            "text": "I now have taco all over my monitor from reading 'Daddy Curle'"
+        },
+        {
+            "user": "FinnT730",
+            "text": "Daemon would be hard to pull off for that though...."
+        },
+        {
+            "user": "DaemonUmbra",
+            "text": "rm -f /bee/illy"
```
Comment (ravenbuilder934) [inline quotes/data.json:2004]: tsk tsk, where's the null for 404?

File: quotes/data.json [hunk 6]
```diff
@@ -1878,6 +1888,197 @@
...
+            "user": "Quartz",
+            "text": "Nah its just happy to see you. Bouncin with joy"
+        },
+        {
+            "user": "Amethyst",
+            "text": "w-w-w-whitespace!"
+        },
+        {
+            "user": "Quartz",
+            "text": "The real hell was in us all along [...] That or i just set fire to everything again"
```
[the remark this answers is not in the document]
Comment (TheCurle) [reply quotes/data.json:1938]: ?

File: quotes/data.json [hunk 7]
```diff
@@ -1878,6 +1888,197 @@
...
+            "text": "Also also why do you treat women as objects?"
+        },
+        {
+            "user": "diesieben07",
+            "text": "QUOTE THAT QUOTE THAT QUOTE THAT"
+        },
+        {
+            "user": "illy",
+            "side_text": "<strong style=\"font-size: 115%\">the insane</strong>",
+            "text": "Those insane people also live in the funny farm [...] The Funny Farm, where life is beautiful all the time, and I'll be happy to see those nice young men in their clean white coats"
```
[the remark this answers is not in the document]
Comment (TheCurle) [reply quotes/data.json:1984]: ?

File: quotes/data.json (hunk 5 above)
Comment (TheCurle) [reply quotes/data.json:2004]: Whoops, didn't see that

File: quotes/data.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (ravenbuilder934) [reply quotes/data.json:1984]: pppp. P p p p p

**Comment by TheCurle** [comment]
SHIP ITTTTT

**Comment by FinnT730** [comment]
\O/

**Pull request merged**